package.json

```diff
@@ -71,7 +71,7 @@
     "lint:js": "eslint --ext .js,.jsx --ignore-pattern public,static .",
     "lint:md": "remark content/posts/",
     "write-good": "write-good $(glob 'content/posts/**/*.md')",
-    "format:js": "prettier '**/*.{js,jsx}' --write"
+    "format:js": "prettier 'src/**/*.{js,jsx}' --write"
```

Comment:
Prettier also prettifies the `.cache` folder. I would say prettifying `src/..` suffices.